docs/source/authentication_with_modelon_impact.rst

```diff
@@ -68,6 +68,13 @@ Revoking the token will cause the prompt to be shown again and a new token needs
 After authenticating against the JupyterHub API, the user must also authenticate against the Modelon Impact server 
 by following the instructions in the section :ref:`below<Modelon Impact API key generation>`.
 
+**Note:**
+The token used to authenticate JupyterHub is looked up in the following preference order:-
+
+1. Token set via interactive prompt

```

Comment:
My thinking was for a user with a interactive flag set to True and no env variable/file available, then interactive is first preference. Also since interactive is None by default(in JH we have it set as True by an env) a user operating outside JH would set this to True the first time. But you are right, about it being described as a fallback makes more sense. I will move interactive to '3'.